app/views/layouts/application.html.erb

```diff
@@ -63,5 +63,17 @@ window.current_page_name = "<%= params[:controller] %>/<%= params[:action] %>";
 
 <%= render "layouts/footer" %>
 
+<% if @queue_stats %>
```

Comment:
Probably just a stray print, but maybe a useful UI element for students.

Interestingly, it's useful in this particular version but not in the external-service version.